app/views/field_test/experiments/show.html.erb

```diff
@@ -0,0 +1,117 @@
+<style>
+  details details { padding: 1em 0 0 1.5em; }
+</style>
+<% experiment = @experiment %>
+<h1><%= experiment.name %><% unless experiment.active? %> <small class="closed">Completed</small><% end %></h1>
```

Comment:
Quick, non-blocking suggestion: let's remove the extra space here if it's not needed.
```suggestion
<h1><%= experiment.name %><% unless experiment.active? %><small class="closed">Completed</small><% end %></h1>
```